examples/src/main/scala/com/criteo/cuttle/examples/HelloWorld.scala

```diff
@@ -23,7 +23,7 @@ object HelloWorld {
       }
 
     val hello2 =
-      Job("hel.lo2", hourly(start), "Hello 2") { implicit e =>
+      Job("hello2", hourly(start), "Hello 2") { implicit e =>
```

Comment:
This has been introduced intentionally as part of https://github.com/criteo/cuttle/pull/148 in order to reproduce a bug with ids containing dots.

Reply:
I know but I don't think the examples should be used for that. Better to introduce test projects in `src/test/` for that.

Reply:
Agreed